raiden/smart_contracts/ChannelManagerLibrary.sol

```diff
@@ -55,10 +56,8 @@ library ChannelManagerLibrary {
     function partyHash(address address_one, address address_two) private constant returns (bytes32) {
         if (address_one < address_two) {
             return sha3(address_one, address_two);
-        } else if (address_one > address_two){
-            return sha3(address_two, address_one);
         } else {
-            revert(); // if the two addresses provided are identical
+            return sha3(address_two, address_one);
```

Comment:
Pointed out by @hackaugusto in rocket chat:
github.com/raiden-network/raiden/blob/master/raiden/smart_contracts/NettingChannelContract.sol#L27